common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/RemoteBlockPushResolverSuite.java

```diff
@@ -1415,6 +1440,24 @@ private void pushBlockHelper(
     }
   }
 
+  private void verifyMetrics(
```

Comment:
Looks like the expectedCachedBlocksBytes are all 0 in all the callers, do we need to add any UTs for testing the CachedBytes metrics?

Reply:
Sure, I enhanced the UTs `testDeferredBufsAreWrittenDuringOnData` and `testDeferredBufsAreWrittenDuringOnComplete` to cover the test for `expectedCachedBlocksBytes` since the UTs produced the cached block bytes. Thanks for pointing this out.